### PR TITLE
Redesigned table to use card ui in tags page

### LINKED ISSUE #5090: Implement new design for Tags page (needs breaking up)
Building on #4922, we should work to rebuild the page at https://publiclab.org/tags following this design:

Screen Shot 2019-03-15 at 12 45 20 PM

There's discussion about this here: https://publiclab.org/notes/edie_blues/01-09-2019/the-tags-page-mockup-let-me-know-what-you-think#c22154

The current template is here: https://github.com/publiclab/plots2/tree/master/app/views/tag/index.html.erb

Steps that could be broken out include:

* [x] making it 2-column #5100
* [x] adding https://publiclab.org/stats/graph in an iframe to the 2nd column
* [x] adding the links below the graph
* [x] Redesigning the search bar (https://github.com/publiclab/plots2/pull/5121)
* [ ] redesigning the table to use a Card ui instead (starting with panels, here: https://getbootstrap.com/docs/3.3/components/#panels) in a template partial

I'd love help with this one!

### PR DESCRIPTION
**Pull request by divyabaid16**
Fixes #5090  (<=== Add issue number here)

Screenshot for the card ui:
Selection_119

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by ananyaarun** [comment]
Wow @divyabaid16 this is a really nice change :) !!

**Comment by divyabaid16** [comment]
Thank you @ananyaarun :smile:
Can you please review this @jywarren ?

**Comment by jywarren** [comment]
This looks super! 🎉 

I'd like to ask for a couple small tweaks - can we unbold the tag titles, and could we add a bit more padding to the bottom of each box? 

Then this should be good to go, and we can do a follow-up with adding more stuff to each tag box, like a few recent posts for example. 

Great work!!!

File: app/views/tag/index.html.erb [hunk 1]
```diff
@@ -26,41 +26,33 @@
...
-                <a rel="tooltip" title="<%= t('tag.show.unfollow') %>" class="btn btn-default btn-sm active" href="/unsubscribe/tag/<%= tag.name %>"><i class="fa fa-user-plus" aria-hidden="true"></i> <%= t('tag.show.following') %></a>
-                <% else %>
-                <a class="btn btn-default btn-sm index-follow-buttons follow-btn-remote" href="/subscribe/tag/<%= tag.name %>" data-remote="true" ><i class="fa fa-user-plus" aria-hidden="true"></i> <%= t('tag.show.follow') %></a>
-                <% end %>
-          </td>
-        <% end %>
-      </tr>
-    <% end %>
-  </table>
+      <div class="panel panel-default">
```
Comment (jywarren) [inline app/views/tag/index.html.erb]: Actually, would you like to try breaking this into its own partial, maybe called `_topicCard.html.erb`?

**Comment by divyabaid16** [comment]
@jywarren I have made the changes. 
The updates screenshot:
Selection_125

Thanks!!

**Comment by jywarren** [comment]
This is looking awesome! It's still a little empty without the content in there. Would you like to try displaying the titles of three of the most recent posts in each topic, in a compact list, in the box? Or do you think we should merge this first and do it in a follow-up? Thank you!!!

**Comment by divyabaid16** [comment]
@jywarren I will try displaying the titles of three of the most recent posts in each topic. The PR was getting very long so I thought that I would do it in the next.
Thanks :)

**Comment by jywarren** [comment]
Thank you, usually i would agree! I just think we're almost there and they
look a little empty right now. Thank you!!!

**Comment by divyabaid16** [comment]
@jywarren I have tried displaying the titles of three of the most recent posts in each topic and this is how it looks now:
Selection_154

Please let me know if there is any further change.
Thanks!

**Comment by jywarren** [comment]
This is beautiful!!! It's perfect! Thank you so much!

**Comment by jywarren** [comment]
Great. I'm going to try publishing it now!!!

**Comment by divyabaid16** [comment]
@jywarren please do tag me if any other help is required as I won't be much available due to my exams.
Thanks!

**Pull request merged**